### PR TITLE
fix: prevent duplicate users in seed task

### PR DESCRIPTION
**Pull request by digioak**
See https://github.com/denoland/saaskit/issues/588

### CODE REVIEW

File: tasks/db_seed.ts [hunk 1]
```diff
@@ -40,17 +40,19 @@ const items = stories.map(({ by: userLogin, title, url, score, time }) => ({
   createdAt: new Date(time * 1000),
 })).filter(({ url }) => url);
 
+const users = Array.from(new Set(items.map((user) => user.userLogin)));
```
Comment (iuioiua) [inline tasks/db_seed.ts:43]:
```suggestion
const users = new Set(items.map((user) => user.userLogin));
```
`Set` is iterable. `Array.from()` is not needed 🙂

**Review by iuioiua — CHANGES_REQUESTED** [review]

File: tasks/db_seed.ts (hunk 1 above)
Comment (digioak) [reply tasks/db_seed.ts]: > `Set` is iterable. `Array.from()` is not needed 🙂

Very true! Got confused because of mapping and filtering earlier on. Fixed.

**Comment by iuioiua** [comment]
Please run `deno fmt`

**Comment by digioak** [comment]
> Please run `deno fmt`

Had to disable my local prettier and eslint.

**Review by iuioiua — APPROVED** [review]
LGTM. Thank you

**Pull request merged**